src/test/java/com/exasol/errorreporting/ErrorMessageBuilderTest.java

```diff
@@ -99,4 +99,175 @@ void testTicketMitigation() {
         assertThat(message, equalTo(
                 "E-ERJ-TEST-1: Something went wrong. This is an internal error that should not happen. Please report it by opening a GitHub issue."));
     }
+
+    @Test
+    void testmessageWithoutParameters() {
```

Comment:
Could you add a test that mixes the two APIs?
(first parameter inline, second using `parameter()`)